apps/backoffice-v2/src/Router/types.ts

```diff
@@ -10,6 +10,7 @@ export type TRouteWithoutChildren = {
 
 export type TRouteWithChildren = {
   text: ReactNode | ReactNode[];
+  href?: string;
```

Comment:
Revert this change. Routes with children will never have an href. Also, having an optional href will leave room for mistakes.